app/src/main/jni/modules/recorder.cpp

```diff
@@ -26,6 +28,41 @@ struct RecordingModule : public AlgorithmModule {
                 (videoRecName.is_null() || !recordCamera) ? "" : videoRecName.get<std::string>());
 
         recorder->setVideoRecordingFps(settings.at("targetFps").get<float>());
+
+        infoFile = !settings.at("infoFileName").is_null() ? settings.at("infoFileName") : "";
+        paramsFile = !settings.at("parametersFileName").is_null() ? settings.at("parametersFileName") : "";
+    }
+    
+    void writeInfoFile(const std::string &mode, const std::string &device) {
+        // TODO: Move implementation to recorder?
+        if (infoFile.empty())
+            return;
+        json infoJson = R"(
+            {
+                "camera mode": "",
+                "device": "",
+                "tags": [
+                    "android"
+                ]
+            }
+        )"_json;
+        infoJson["camera mode"] = mode;
+        infoJson["device"] = device;
+        std::ofstream fileOutput(infoFile);
+        std::ostream &output(fileOutput);
```

Comment:
This line should not be needed, just `fileOutput << infoJson.dump() << std::endl;`